docker/BASE

```diff
@@ -0,0 +1,23 @@
+# `shinsaku0523/zkevm` base image of zkevm
+# rust: nightly-2021-11-17
+# go:   1.16
+FROM ubuntu:16.04
```

Comment:
If users have a different system than Ubuntu, this will not work I assume right? Is there anything that we can do to make it more generic?

Reply:
> If users have a different system than Ubuntu, this will not work I assume right?

It works if user don't use `alpine` but `CentOs`, `Ubuntu` or `MacOS`.
If we make `alpine` image, we can provide almost every environment but to create `alpine` image seems so difficult I think.

Reply:
kk if the major distros are supported I'm happy!! :)

Do you know what happens with Arch?

Reply:
> Do you know what happens with Arch?

It seemed the memory allocation problem, and the error said `(signal: 11, SIGSEGV: invalid memory reference)`.
I searched about that error but I couldn't understand what heppened.
It's depending on `opt-level`, llvm and lto so it didn't fail when `opt-level` is not 3.
I think it's rust compiler problem or something.
Should I create other OS images?

Reply:
btw why not using a later version of ubuntu? 16.04 is quite old. we can try to use 20.04